README.md

```diff
@@ -105,8 +108,9 @@ using a GraphQL API.
 Take the following steps to set up Juicebox's subgraph for local development:
 
 1. Join [Peel's discord server](https://discord.gg/akpxJZ5HKR).
-2. Go to [the dev channel](https://discord.com/channels/939317843059679252/939705688563810304)
-    and inquire about mainnet and rinkeby subgraph URLs.
+2. Go to
```

Comment:
I was curious why this diff happened. Turns out we're not linting staged markdown files. My editor formats on save, but everyone else's might not be configured to do the same.

Going to add a diff to address `lint-staged` configuration to include `.md` files.

Reply:
I have addressed:
- lint staged in #1049 
- run prettier and add a yarn script in #1050